src/ports/postgres/modules/graph/apsp.sql_in

```diff
@@ -412,9 +412,9 @@ m4_ifdef(`\_\_HAS_FUNCTION_PROPERTIES\_\_', `MODIFIES SQL DATA', `');
 
 CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.graph_apsp_get_path(
     apsp_table             TEXT,
-    source_vertex          INT,
-  dest_vertex            INT,
-  path_table             TEXT
+    source_vertex          BIGINT,
```

Comment:
will this and other such changes be backwards compatible ?

Reply:
Yes, we still have tests that pass integer columns to these functions that work as expected.